src/ui/components/UserCollection/index.js

```diff
@@ -0,0 +1,73 @@
+/* @flow */
+import invariant from 'invariant';
+import * as React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import LoadingText from 'ui/components/LoadingText';
+import type { I18nType } from 'core/types/i18n';
+
+type Props = {|
+  authorUsername?: string,
+  id: number,
+  loading?: boolean,
+  name?: string,
+  numberOfAddons?: number,
+  slug?: string,
+|};
+
+type InjectedProps = {|
+  i18n: I18nType,
+|};
+
+type InternalProps = { ...Props, ...InjectedProps };
+
+export const UserCollectionBase = (props: InternalProps) => {
+  const {
+    authorUsername,
+    id,
+    loading,
+    name,
+    numberOfAddons,
+    slug,
+    i18n,
+  } = props;
+
+  let href;
+  let numberText;
+
+  if (!loading) {
+    invariant(authorUsername, 'authorUsername is required');
+    invariant(name, 'name is required');
+    invariant(slug, 'slug is required');
+    invariant(
+      Number.isInteger(numberOfAddons),
+      'numberOfAddons must be a number',
+    );
+
+    href = `/collections/${authorUsername}/${slug}/`;
+    numberText = i18n.sprintf(
+      i18n.ngettext('%(total)s add-on', '%(total)s add-ons', numberOfAddons),
+      { total: i18n.formatNumber(numberOfAddons) },
```

Comment:
For some reasons, RTL langs return a sort of dot when there are 0 add-ons in the collection:

![screen shot 2018-06-25 at 14 54 07](https://user-images.githubusercontent.com/217628/41851295-b16930fc-7887-11e8-9945-c653208b6be0.png)


I am not sure if it is expected or not.

Reply:
I believe that the dot is just what is output for the language for a 0, but I'm not 100% certain of that. It seems to be consistent though.

Reply:
Yes, I was wondering.. I *think* we can probably leave it as is.